### PR TITLE
Data update: 2021_02_16_04_37_27

### PR DESCRIPTION
**Pull request by beets**
Updated Covid, FBI and Unemployment data

It looks like we might need to add a secondary key for our ranking sort, the diffs there are due to instability when there is a similar value across places.

### CODE REVIEW

**Review by pradh — APPROVED** [review]

File: test/integration/golden_response/staging/query/statvar-obs.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     {
       "cells": [
         {
-          "value": "dc/o/syrpc3m8q34z7"
+          "value": "dc/o/bqtfmc351v0f2"
```
Comment (shifucun) [inline test/integration/golden_response/staging/query/statvar-obs.json:10]: this diff would get stable after https://github.com/datacommonsorg/mixer/pull/467

**Review by shifucun — APPROVED** [review]

File: test/integration/golden_response/staging/get_stat_all/result.json [hunk 2]
```diff
@@ -58,7 +58,9 @@
                 "2014": 9433470,
                 "2015": 9185231,
                 "2016": 9161163,
-                "2017": 8940603
+                "2017": 8940603,
+                "2018": 8403390,
```
Comment (hanlu09205) [inline test/integration/golden_response/staging/get_stat_all/result.json:62]: This looks right to me, as we are adding 2018/2019 FBI country level data.

File: test/integration/golden_response/staging/get_stat_set/latest.json [hunk 3]
```diff
@@ -30,8 +30,8 @@
       "stat": {
         "country/FRA": {},
         "country/USA": {
-          "date": "2017",
-          "value": 8940603,
+          "date": "2019",
+          "value": 8132863,
```
Comment (hanlu09205) [inline test/integration/golden_response/staging/get_stat_set/latest.json:34]: Ditto.

**Review by hanlu09205 — APPROVED** [review]

**Comment by beets** [comment]
thanks all!

**Pull request merged**